src/app/features/report/report/report.component.html

```diff
@@ -111,7 +111,7 @@
           <div class="text-gray-600">Ratio of suspicious records in the batch in respect to training data:</div>
           <div class="font-bold ml-2">{{report.batchStats.susRatio | number:'1.1-1'}}</div>
         </div>
-        <hs-button kind="stroked" color="primary" label="See profiler" [routerLink]="['/', 'models', modelName$ | async, modelVersion$ | async, 'profiler']"></hs-button>
+        <hs-button kind="stroked" color="primary" label="See profiler" [routerLink]="['/', 'models', modelName$ | async, modelVersion$ | async, 'profiler', encode(report.file)]"></hs-button>
```

Comment:
Create pipe for `encode(report.file)`, and use it like that `report.file | base64`